yellowbrick/target/feature_correlation.py

```diff
@@ -0,0 +1,87 @@
+# yellowbrick.classifier.feature_correlation
+# Class balance visualizer for showing per-class support.
```

Comment:
Thanks for adding our code header! I see what you mean by your question about the ID line now. Here, the first line should be the module path, e.g. `yellowbrick.target.feature_correlation` and the second line should be a quick description, here `"Feature Correlation to Dependent Variable Visualizer."` 

Everything else looks good, I've already mentioned the ID line, but you don't need to add my name as Author -- I can do that if I update this file. Also note that if you make changes to a file in YB, feel free to add yourself to the author line in the ID tag.